collab-document/tests/importer/md_importer_test.rs

```diff
@@ -453,3 +453,33 @@ fn test_table() {
     }
   }
 }
+
+#[test]
+fn test_aside() {
+  let markdown = r#"<aside>
```

Comment:
It's the HTML tag, right? I tried to find if there's `parse_options` in `mdast` to enable parsing the callout, unfortunately, found nothing.

Reply:
yes.